test/unit/attributes.js

```diff
@@ -1523,17 +1565,22 @@ QUnit.test( "option value not trimmed when setting via parent select", function(
 	assert.equal( jQuery( "<select><option> 2</option></select>" ).val( "2" ).val(), "2" );
 } );
 
-QUnit.test( "Insignificant white space returned for $(option).val() (#14858)", function( assert ) {
-	assert.expect( 3 );
+QUnit.test( "Insignificant white space returned for $(option).val() (#14858, gh-2978)", function( assert ) {
+	assert.expect( 16 );
 
 	var val = jQuery( "<option></option>" ).val();
 	assert.equal( val.length, 0, "Empty option should have no value" );
 
-	val = jQuery( "<option>  </option>" ).val();
-	assert.equal( val.length, 0, "insignificant white-space returned for value" );
+	jQuery.each( [ " ", "\n", "\t", "\f", "\r" ], function() {
+		var val = jQuery( "<option>" + this + "</option>" ).val();
```

Comment:
[Not](https://github.com/jquery/jquery/blob/master/src/serialize.js#L74) [everywhere](https://github.com/jquery/jquery/blob/master/src/attributes/val.js#L154), but I think I agree that naming what I'm after is better.